src/odemis/gui/comp/positions.py

```diff
@@ -0,0 +1,254 @@
+import logging
+import math
+import wx
+from odemis import gui, model
+from odemis.gui.comp.text import UnitFloatCtrl
+
+
+class EditMeteorPositionsDialog(wx.Dialog):
+    """Panel for METEOR Stage Positions"""
+
+    def __init__(self, parent, stage_md: dict):
+        wx.Panel.__init__(self, parent=parent)
+        self._parent = parent
+        self._update_pressed = False
+
+        self.SetForegroundColour(gui.FG_COLOUR_EDIT)
+        self.SetBackgroundColour(gui.BG_COLOUR_MAIN)
+        self.SetTitle("Edit METEOR Positions")
+
+        self.main_sizer = wx.BoxSizer(wx.VERTICAL)
+        self.SetSizer(self.main_sizer)
+
+        self.gb_sizer = wx.GridBagSizer()
+        self.num_rows = 0
```

Comment:
This doesn't need to be an attribute. It could be just a variable (discarded after the end of `__init__`).